builder/amazon/ebssurrogate/step_snapshot_volumes.go

```diff
@@ -61,6 +64,7 @@ func (s *StepSnapshotVolumes) Run(ctx context.Context, state multistep.StateBag)
 	ui := state.Get("ui").(packer.Ui)
 
 	s.snapshotIds = map[string]string{}
+	s.snapshotMutex = sync.RWMutex{}
```

Comment:
A mutex does not need to be initialised.

```suggestion
	s.snapshotMutex = sync.RWMutex{}
```